backend/src/main/java/com/myshop/utils/variable/ConfigurationVariable.java

```diff
@@ -0,0 +1,12 @@
+package com.myshop.utils.variable;
+
+public class ConfigurationVariable {
+    public static final String SECRET = "Secret";
+    public static final int EXPIRED_ACCESS_TIME = 10 * 60 * 1000;
+    public static final int EXPIRED_REFRESH_TIME = 30 * 60 * 1000;
+    public static final String DEFAULT_SIZE = "20";
```

Comment:
why do we use string declare for a variable using integer value?

Reply:
- Because in DEFAUL_VALUE, it requires a string not a integer. 
![image](https://user-images.githubusercontent.com/83630874/186125399-a63abafd-08ed-43a3-8549-2bc8f950724c.png)
- If I use integer, it has a caution 
![image](https://user-images.githubusercontent.com/83630874/186125765-fd7a1c31-01c9-4bcd-8dfb-fca0c66a77c3.png)